build/params_testground.go

```diff
@@ -49,13 +49,13 @@ var (
 	TicketRandomnessLookback     = abi.ChainEpoch(1)
 	WinningPoStSectorSetLookback = abi.ChainEpoch(10)
 
-	TotalFilecoin     uint64 = 2_000_000_000
-	MiningRewardTotal uint64 = 1_400_000_000
+	FilBase               uint64 = 2_000_000_000
+	FilAllocStorageMining uint64 = 1_400_000_000
```

Comment:
Y'all should really import these values from the reward actor, else the actual reward emission schedule will not match whatever you use these for. E.g. maybe you won't give the reward actor enough balance.